qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/ModelIT.java

```diff
@@ -142,7 +144,7 @@ public void testKNNModelDefault() throws Exception {
 
     // KNN Delete Model test for model in Training State
     public void testDeleteTrainingModel() throws Exception {
-        byte[] testModelBlob = "hello".getBytes();
+        byte[] testModelBlob = "hello".getBytes(StandardCharsets.UTF_8);
```

Comment:
we do have multiple places with a method calls without Charset (e.g. https://github.com/opensearch-project/k-NN/blob/main/src/test/java/org/opensearch/knn/plugin/transport/GetModelResponseTests.java#L36), why we've changed only this one?

Reply:
This came as a forbidden api, so I fixed it.

Reply:
Ohh...I didn't run into this issue when I made the changes. Not sure if OpenSearch core made these changes after I raised the PR